docs/features.md

```diff
@@ -152,85 +152,51 @@ donejs deploy
 
-### Supports All Browsers, Even IE8
+Many people don't care about this because IE8 is on its way out, which is a very good thing! 
 
-DoneJS is compatible with modern and legacy versions of all browsers, including Chrome, Firefox, Safari, and Internet Explorer back to version 8. While other frameworks eschew legacy browser support, DoneJS embraces it. Doing so maximizing compatibility, and thus potential audience adoption, of your app.
+But its not quite dead yet, at still over 3% of the browser market in September 2015. For many mainstream websites, banks, and ecommerce applications, IE8 continues to hang around the browser stats. 
 
-[As of September 2015](http://gs.statcounter.com/#browser_version_partially_combined-ww-monthly-201508-201509-bar), 8.77% of Internet users are not using the latest version of their web browser. Of those, 2.26% still use Internet Explorer 8 as their primary browser! 
+And while other frameworks like AngularJS and EmberJS don't support IE8, DoneJS makes it easy to write one app that runs everywhere.
 
-<img src="./static/img/browser-stats.png" alt="Source: http://gs.statcounter.com/#browser_version_partially_combined-ww-monthly-201508-201509-bar" />
-_Source: http://gs.statcounter.com/#browser_version_partially_combined-ww-monthly-201508-201509-bar_
+### Real Time Connected
 
-Who are these users? Most are corporate users tethered to an IT policy, or users in emerging markets. Both represent potentially large market segments, and support for them should be considered when building your application.
+DoneJS is designed to make it easy to build applications that connects users in real-time.
 
-By maintaining legacy browser support, DoneJS enables you to maximize the reach of your audience with no additional effort.
+[Socket.io](https://socket.io) provides the basics to add real-time capabilities to any JavaScript application, but the challenge of integrating real-time updates into your code remains. DoneJS makes weaving Socket.io backends into your UI simple and automatic.
 
-### Real Time Connected
+#### How it works
 
-Building apps that require real time updates and communication using DoneJS is easy. DoneJS can seemlessly integrate with your [Socket.io](https://socket.io) backend. After configuring Socket.io on the client to use can-connect, lists and indiviual models will be updated in real time. Your views will automatically update to reflect the changes.
+<video controls style='width:100%;' poster="http://connect.canjs.com/assets/videos/Can-Connect-Anim-2.s3b1-720p.png" src="http://connect.canjs.com/assets/videos/Can-Connect-Anim-2.s3b1-720p.mp4"></video>
 
-<img src="./static/img/donejs-real-time-v2.gif" alt="How a real time connected app works!" />
+DoneJS' model layer users set logic to maintain lists of data represented by certain properties, like all todos that are due tomorrow. These lists are rendered to the UI via data bound templates.
```

Comment:
"all todos that are due tomorrow" doesn't really connect to "lists of data represented by certain properties".